app/javascript/listings/components/ListingTagsField.jsx

```diff
@@ -0,0 +1,116 @@
+import { h } from 'preact';
+import { useEffect, useMemo, useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { useTagsField } from '../../hooks/useTagsField';
+import { TagAutocompleteOption } from '../../crayons/MultiSelectAutocomplete/TagAutocompleteOption';
+import { TagAutocompleteSelection } from '../../crayons/MultiSelectAutocomplete/TagAutocompleteSelection';
+import { MultiSelectAutocomplete } from '@crayons';
+
+export const ListingTagsField = ({
+  defaultValue,
+  onInput,
+  categorySlug,
+  name,
+  onFocus,
+}) => {
+  const listingState = useMemo(
+    () => ({
+      additionalTags: {
+        jobs: [
+          'remote',
+          'remoteoptional',
+          'lgbtbenefits',
+          'greencard',
+          'senior',
+          'junior',
+          'intermediate',
+          '401k',
+          'fulltime',
+          'contract',
+          'temp',
+        ],
+        forhire: [
+          'remote',
+          'remoteoptional',
+          'lgbtbenefits',
+          'greencard',
+          'senior',
+          'junior',
+          'intermediate',
+          '401k',
+          'fulltime',
+          'contract',
+          'temp',
+        ],
+        forsale: ['laptop', 'desktopcomputer', 'new', 'used'],
+        events: ['conference', 'meetup'],
+        collabs: ['paid', 'temp'],
+      },
+    }),
+    [],
+  );
+
+  const { defaultSelections, fetchSuggestions, syncSelections } = useTagsField({
+    defaultValue,
+    onInput,
+  });
+  const [suggestedTags, setSuggestedTags] = useState([]);
+
+  useEffect(() => {
+    // Push in this way: { name: 'remote' }
+    const categorySuggestedTags = (
+      listingState.additionalTags[categorySlug] || []
+    ).map((name) => ({ name }));
+    setSuggestedTags(categorySuggestedTags);
+  }, [listingState, categorySlug]);
+
+  const fetchSuggestionsWithAdditionalTags = async (searchTerm) => {
+    const fetchedSuggestions = await fetchSuggestions(searchTerm);
+    const suggestedNames = fetchedSuggestions.map((t) => t.name);
+
+    // Search in the suggestedTags array
+    const additionalItems = suggestedTags.filter(
+      (t) => t.name.startsWith(searchTerm) && !suggestedNames.includes(t.name),
+    );
+    // Join fetched and additional items
+    const suggestionsResult = [...fetchedSuggestions, ...additionalItems];
+    // Order suggestionsResult by name
+    suggestionsResult.sort((a, b) => {
+      if (a.name < b.name) return -1;
+      if (a.name > b.name) return 1;
+      return 0;
+    });
```

Comment:
```suggestion
    suggestionsResult.sort((a, b) => ( a.name > b.name ) ? 1 : -1);
```
(not blocking) - I think we could simplify this, but I don't mind either way 🤷‍♀️

Reply:
I've found this way and works good:
```js
suggestionsResult.sort((a, b) => a.name.localeCompare(b.name));
```
What do you think?

Reply:
sounds great - and looks like it's well supported across browsers 🙌

Reply:
Nice! I will use it